golang1.17/lib/launcher.go

```diff
@@ -87,8 +87,7 @@ func main() {
 		}
 		output, err := execute(Main, inbuf)
 		if err != nil {
-			fmt.Fprintf(out, `{"error":%q}`, err.Error())
-			continue
+			output = []byte(fmt.Sprintf(`{"error":%q}`, err.Error()))
```

Comment:
is `output` guaranteed empty if `err` is not `nil` since this overwrites `output`?

Reply:
I think we're returning `nil, err` everywhere, so I think that's the invariant we're after, yeah.